src/phpDocumentor/Pipeline/Stage/Transform.php

```diff
@@ -89,8 +97,8 @@ public function __invoke(Payload $payload): Payload
         $transformations = $templates->getTransformations();
 
         foreach ($project->getVersions() as $version) {
-            $apiSets = $version->getDocumentationSets()->filter(ApiSetDescriptor::class);
```

Comment:
I have changed this to remove the filter because the transformer should either be fully ApiSet specific and we need to make big changes to the pipelines; or it should be agnostic and the individual writers should check if they are suited to a specific set of documentation.

At this moment, I am choosing the latter. But I think I need to revisit some of the writers to actually add such checks

Reply:
Done, I have added extra checks to the individual writers